CHANGES.rst

```diff
@@ -7,6 +7,11 @@ Fixes
 
 - Fixed capitalization of ``Shards`` tab label
 
+- Updated keywords list so that they are recognised and painted in red.
+
+- Upgraded ``node-sass``.
```

Comment:
If we add such dependency upgrades to the `CHANGES` we usually also include a reason. What is the effect of the upgrade for an end-user?